tuxemon/npc.py

```diff
@@ -754,6 +754,49 @@ def remove_monster(self, monster: Monster) -> None:
             self.monsters.remove(monster)
             self.set_party_status()
 
+    def evolve_monster(self, monster: Monster, evolution: str) -> None:
+        """
+        Evolve a monster from this npc's party.
+
+        Parameters:
+            monster: Monster to remove from the npc's party.
+            evolution: Monster to add to the npc's party.
+
+        """
+        if monster not in self.monsters:
+            return
+        # Store old monster data
+        old_slot = self.monsters.index(monster)
+        old_level = monster.level
+        old_current_hp = monster.current_hp
```

Comment:
You don't really need variables for all of these, but not a big deal.

Reply:
Look, tell me what I need to remove

Reply:
everything old?